docs/_data/product/atomic-definitions.yml

```diff
@@ -222,31 +222,46 @@ width-static:
 width-max:
   - class: "wmx-initial"
     output: "max-width: initial;"
-    responsive: false
+    responsive: true
   - class: "wmx1"
     output: "max-width: 105px;"
     responsive: false
   - class: "wmx2"
     output: "max-width: 211px;"
     responsive: false
+  - class: "wmx25"
+    output: "max-width: 25%;"
+    responsive: false
   - class: "wmx3"
     output: "max-width: 316px;"
     responsive: false
+  - class: "wmx33"
+    output: "max-width: 33%;"
```

Comment:
I was thinking about values like `33%` and `66%` – is having them that common? Having three elements with width `33%` means that there will be `1%` remaining. It's usually not a lot, but it feels to me like an icon with blurred pixels :D 

A bit crazy, but how about using `calc(100% / 3)` instead (and `calc(100% / 3 * 2)` for 66%)?

Reply:
Perhaps we don't need thirds at all, but I think relying on calc would totally work in this case. I threw these in there for the conversation, but I'm leaning away from them, especially since our grid has things like `grid__allcells4` and the like. 🤔 

@hellohynes @balpha Y'all have a strong opinion on keeping 1/3 and 2/3 percentage widths? I'm leaning toward just dropping them off, especially since we don't have `w33`. Oops!

Reply:
No strong opinions here, happy with dropping them.